### DISCUSSION TITLE
Guarantee of affine status of `Mat4::look_at_rh()` and `Mat4::look_at_lh()` output

### DISCUSSION THREAD

**Discussion by ClydeHobart** [bitshifter/glam-rs#310] [Q&A]
I've been getting some asserts when I turn on `debug-glam-assert` from calling `transform_point3()` on the output of `Mat4::look_at_rh()`, but upon inspection, row 3 typically falls in the range [`[-0.0, -0.0, -0.0, 1.0]`, `[0.0, 0.0, 0.0, 1.0000001]`]. There's no means of supplying a perspective to these `look_at_*` functions, so shouldn't the error introduced by writing `[0.0, 0.0, 0.0, 1.0]` to row 3 be negligible? I'm admittedly not the most knowledgeable on 4x4 matrices, so apologies if I'm making baseless assumptions.

**Comment by bitshifter** [comment]
Yeah it looks like that assert needs an epsilon, it's doing an exact comparison with `[0, 0, 0, 1]`. I'll make a bug for this.

**Comment by bitshifter** [comment]
I've pushed a fix for this to main. I'm not sure when I'll make a new release, want to get some other stuff in as well.

**Reply by ClydeHobart** [reply]
Thanks!
